### PR TITLE
Create WilsonChen.txt

### PR DESCRIPTION
**Pull request by wilsonchen02**
a

### CODE REVIEW

File: introductions/WilsonChen.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name: Wilson Chen
+Pronouns: He/him
+Year: Junior
+Major: Computer Engineering
+
+I started programming in high school and have taken EECS 280, 281, and currently taking 370.
```
Comment (WinsonChen114) [inline introductions/WilsonChen.txt]: and am currently*

File: introductions/WilsonChen.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+I have experience in Java, C/C++, Python and I do not like euchre.
+
+Outside of school/MRover, I play a lot of different games and watch the Urban Rescue Ranch channel on youtube (go watch it!!).
```
Comment (WinsonChen114) [inline introductions/WilsonChen.txt]: capitalize Youtube

**Review by WinsonChen114 — CHANGES_REQUESTED** [review]
See comments :)

File: introductions/WilsonChen.txt (hunk 1 above)
Comment (wilsonchen02) [reply introductions/WilsonChen.txt:6]: or am i?

**Review by WinsonChen114 — APPROVED** [review]
# Epic Games

**Comment by wilsonchen02** [comment]
> # Epic Games

nice!

**Review by tabiosg — APPROVED** [review]
I also do not like euchre!!

**Pull request merged**